xclim/testing/tests/test_indices.py

```diff
@@ -1053,27 +1053,39 @@ class TestJetStreamIndices:
     da_ua = xr.DataArray(
         np.random.rand(66, 3, 5, 5),
         coords={
-            "time": time_coords,
-            "plev": [75000, 85000, 100000],
-            "lon": [120, 121, 122, 123, 124],
-            "lat": [15, 16, 17, 18, 19],
+            "T": time_coords,
+            "Z": [75000, 85000, 100000],
+            "X": [120, 121, 122, 123, 124],
+            "Y": [15, 16, 17, 18, 19],
         },
-        dims=["time", "plev", "lon", "lat"],
+        dims=["T", "Z", "X", "Y"],
         attrs={
             "standard_name": "eastward_wind",
             "units": "m s-1",
         },
     )
-    da_ua.plev.attrs["units"] = "Pa"
-    da_ua.lat.attrs["units"] = "degrees"
+
+    da_ua.Z.attrs = {"units": "Pa", "standard_name": "pressure"}
+    da_ua.X.attrs = {"units": "degrees_east", "standard_name": "longitude"}
+    da_ua.Y.attrs = {"units": "degrees_north", "standard_name": "latitude"}
+    da_ua.T.attrs = {"standard_name": "time"}
```

Comment:
`T` is not being noticed by cf_xarray as `time`

Reply:
That's one assumption we are ok to make in xclim. The time coordinate is always "time", so it can be accessed directly as such `da_ua.time`, no need to go through `cf_xarray` for this one.